package.json

```diff
@@ -30,7 +30,7 @@
   },
   "peerDependencies": {
     "bedrock": "1.12.1 - 3.x",
-    "bedrock-account": "2.x - 4.x",
+    "bedrock-account": "2.x - 5.x",
```

Comment:
The `limit` option that was part of the public API here is being removed which means this will be a breaking change in this module as well (major release).  Also, folks using the 4.x version of bedrock-account may have built systems around the idea of having multiple accounts per email.  So, the upcoming release of this module will *only* work with bedrock-account@5 and will not be backwardly compatible with any prior version, which means that this range should be removed.

Reply:
Updated here d15f518